File: app/src/main/java/com/example/mindlink_celular/MainActivity.java
package com.example.mindlink_celular;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button button = findViewById(R.id.btnIraLogin);

        // Establecer el OnClickListener
        button.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                // Este método se ejecutará cuando se haga clic en el botón

                // Crear un Intent para redireccionar al login
                Intent intent = new Intent(MainActivity.this, Login.class);
                startActivity(intent);
            }
        });
    }


}

File: app/src/main/java/com/example/clinica_psicologicaa/MainActivity.kt
package com.example.clinica_psicologicaa

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.ImageView
import androidx.navigation.ui.AppBarConfiguration
import com.bumptech.glide.Glide

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        showGif()

        val appBarConfiguration = AppBarConfiguration(setOf(R.id.calendarioView,R.id.navigation_dashboard,R.id.navigation_home,R.id.navigation_notifications))

    }
    fun showGif(){
        val gifImageView = findViewById<ImageView>(R.id.splashscreen)
        Glide.with(this)
            .asGif()
            .load(R.drawable.splashscreem)
            .into(gifImageView)
    }
}
